### PR TITLE
Update Muddle pretty printer.

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
Updates from @Rick-Shiffman.

### CODE REVIEW

File: src/rrs/pprint.3 [hunk 1]
```diff
@@ -295,7 +295,30 @@ PP	;"OBLIST"
...
+					   <- .AVSP
+					      3
+					      <FLATSIZE <1 .ML> 99999999>>>
+				      3>>
+			 <SET ML <2 .ML>>)
+			(ELSE <RETURN T>)>>)>>
+
+
+
+;<DEFINE FORMAHEAD (ML "AUX" (AVSP <- <LINLNT .OUTCHAN> <LINPOS .OUTCHAN> .M>))
```
Comment (larsbrinkhoff) [inline src/rrs/pprint.3:321]: The old FORMAHEAD seems to be left in place, commented out.

Comment (Rick-Shiffman) [reply src/rrs/pprint.3:321]: Yes, that's true. These changes correct FORMAHEAD hanlding of an empty FORMs

In FORMAHEAD to produce better looking indentation in PPRINT's output,
306:		  ;"<COND (<L? <LENGTH .ML> 2> <RETURN #FALSE ()>) WAS"
307: 		  <COND (<L=? <LENGTH .ML> 2> <RETURN #FALSE ()>)
308:			(<NOT <==? <TYPE <2 .ML>> FORM>> <RETURN #FALSE ()>)

Changes to:
306:		  <COND (<L? <LENGTH .ML> 2> <RETURN #FALSE ()>)
307: 		  ;<COND (<L=? <LENGTH .ML> 2> <RETURN #FALSE ()>)
308:			(<NOT <==? <TYPE <2 .ML>> FORM>> <RETURN #FALSE ()>)

RRS

**Comment by larsbrinkhoff** [comment]
I updated this pull request.  See the diff here:  
https://github.com/PDP-10/its/compare/9c541614b48ce8c3574287d3e957ce21acf3b416..cc2de3114428671737a9b900a76fb0324470e22b

Is this ok to push out now?

**Comment by Rick-Shiffman** [comment]
Thanks Eric

**Review by eswenson1 — APPROVED** [review]

**Pull request merged**